src/content/plugins/module-concatenation-plugin.md

```diff
@@ -10,12 +10,13 @@ related:
 
 In the past, one of webpack’s trade-offs when bundling was that each module in your bundle would be wrapped in individual function closures. These wrapper functions made it slower for your JavaScript to execute in the browser. In comparison, tools like Closure Compiler and RollupJS ‘hoist’ or concatenate the scope of all your modules into one closure and allow for your code to have a faster execution time in the browser.
 
-This plugin will enable the same concatenation behavior in webpack.
+This plugin will enable the same concatenation behavior in webpack. It is [enabled by default in production](/configuration/optimization/#optimization-concatenatemodules) but you can also manually add it:
```

Comment:
Uhmm, it is better to avoid plugin duplication.

Reply:
One more reason to document the default :-)
What kind of wording would you prefer?
"It is enabled by default in production but you can add it manually if you need it in other circumstances"?

Reply:
"...in other mode".